doc/cli/vast-export.md

```diff
@@ -5,4 +5,27 @@ expression. The export format must be explicitly specified:
 vast export [options] <format> [options] <expr>
 ```
 
-The `export` command is the dual to the `import` command.
+This is easiest explained on an example:
+
+```
+vast export --max-events=100 --continuous json '#timestamp < 1 hour ago'
+```
```

Comment:
In addition to an example invocation, it would also be nice to have *output* examples here.